enos/modules/solo_worker/main.tf

```diff
@@ -0,0 +1,178 @@
+terraform {
+  required_providers {
+    enos = {
+      source = "app.terraform.io/hashicorp-qti/enos"
+    }
+  }
+}
+
+locals {
+  tags = merge(
+    var.common_tags,
+    {
+      "PetName" = random_pet.worker.id
```

Comment:
What is this used for? Does `pet` stand for something?

Reply:
I'll remove it, I use this to more easily identify which pieces of virtual hardware belong to a given instance of this module